docker/0xae02c7b1c324a8d94a564bc8d713df89eae441fe.json

```diff
@@ -0,0 +1,15 @@
+{
+  "mosaicConfigFilePath":"~/.mosaic/dev/mosaic.json",
```

Comment:
```suggestion
  "mosaicConfigFilePath":"~/.mosaic/dev-origin/mosaic.json",
```

Reply:
why dev-origin and not dev?

Reply:
Because mosaic config is stores in `~/.mosaic/dev-origin` directory.